server/datastore/mysql/testing_utils.go

```diff
@@ -17,17 +17,12 @@ import (
 
 const (
 	schemaDbName = "schemadb"
-	dumpfile     = "dump.sql"
+	dumpfile     = "/tmpfs/dump.sql"
 	testUsername = "root"
 	testPassword = "toor"
 	testAddress  = "localhost:3307"
 )
 
-func init() {
-	// Initialize the schema once for the entire test run.
-	initializeSchemaOrPanic()
-}
```

Comment:
@zwass I think this PR needs to be merged before we tag 4.1.0 because I just realized that it breaks the fleet cmd in a non dev environment. Moving this to TestMain fixes it though.